server/src/alda/now.clj

```diff
@@ -93,8 +93,9 @@
    evaluating alda.lisp code, e.g. (score (part 'piano' (note (pitch :c)))),
    or an atom referencing such a map."
   [score & [play-opts]]
-  (sound/with-play-opts (merge {:async? true} play-opts)
-    (sound/play! (if (instance? clojure.lang.Atom score)
-                   @score
-                   score))))
+  (println "testing alda dev setup"))
```

Comment:
It works! 3 moving parts ain't so bad! Thank you, I'll get deving.

![2016-09-06-233701_1334x765_scrot](https://cloud.githubusercontent.com/assets/114097/18293483/e6c83e08-748a-11e6-80df-1c59d429fdc3.png)